{
  "comments": [
    {
      "key": {
        "uuid": "445030b7_fbaf9283",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 9,
      "author": {
        "id": 1000153
      },
      "writtenOn": "2021-09-08T21:42:19Z",
      "side": 1,
      "message": "50/72 wrapping please.",
      "range": {
        "startLine": 9,
        "startChar": 0,
        "endLine": 9,
        "endChar": 51
      },
      "revId": "4e2f4613e5bbcc4fd0390a9f55d53f9114639be0",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "db318918_3283e308",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 13,
      "author": {
        "id": 1000153
      },
      "writtenOn": "2021-09-08T21:42:19Z",
      "side": 1,
      "message": "Is there any details/a link to details around why it\u0027s going to start requiring get_errno?",
      "revId": "4e2f4613e5bbcc4fd0390a9f55d53f9114639be0",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4",
      "unresolved": true
    }
  ]
}